tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -15,6 +16,30 @@
 
 
 class ImportCluster(flows.BaseFlow):
+    def _probe_and_mark_provisioner(self, node_list, integration_id):
+        for node in node_list:
+            try:
+                tags = json.loads(
+                    NS._int.client.read(
+                        "nodes/%s/NodeContext/tags" % node
+                    ).value
```

Comment:
Create NS.tendrl.objects.NodeContext(node_id=node) instead of all these raw read writes